go.mod

```diff
@@ -10,6 +10,10 @@ replace github.com/spf13/viper => github.com/istio/viper v1.3.3-0.20190515210538
 
 replace github.com/docker/docker => github.com/docker/engine v1.4.2-0.20191011211953-adfac697dc5b
 
+replace github.com/envoyproxy/go-control-plane => gitlab.alibaba-inc.com/AliMesh/go-control-plane v0.9.2-0.20200218115735-e6560592fa4e
+
+replace istio.io/api => gitlab.alibaba-inc.com/one-mesh/api v0.0.0-20200215150939-aa28515cd791
+
```

Comment:
Don't care about any dependency starts with "alibaba-inc.com". I'll change it to opensource version before merging and change them to open source version once the changes are applied in opensource libs.